vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SpaceManager.java

```diff
@@ -179,6 +180,13 @@ public ISpace getFullyLoadedSpace(String id) {
         return spaces;
     }
 
+    @Override
+    public List<ISpace> getSpacesWithStatus(SpaceStatus status) {
```

Comment:
still needs a test case

Reply:
The changes for space sidebar are as part of VSPC-48 using which the spaces with space status can be tested.

Reply:
should I implement a java test case?

Reply:
yes, needs a java unit test